docs/topics/gradle/gradle-configure.md

```diff
@@ -0,0 +1,745 @@
+[//]: # (title: Configure your existing project)
+
+To build a Kotlin project with [Gradle](https://docs.gradle.org/current/userguide/getting_started.html), 
+you'll need to add the [Kotlin Gradle plugin](#apply-the-plugin) to your build script file `build.gradle(.kts)` 
```

Comment:
This is the new intro (the first sentence existed but I added more details to it). Everything ahead is old except the "What's next" section.